test/include/RAJA_test-forall-execpol.hpp

```diff
@@ -81,8 +81,6 @@ using OpenMPForallAtomicExecPols =
 #if defined(RAJA_TEST_EXHAUSTIVE)
               , RAJA::omp_parallel_for_static_exec< >
               , RAJA::omp_parallel_for_static_exec<4>
-              , RAJA::omp_parallel_for_nowait_static_exec< >
```

Comment:
why are these exec policies removed from the list?

Reply:
For some reason, they no longer exist as valid policies.

Reply:
Actually, these tests are already covered in lines 58-59, and in the `forall/region` tests. We can safely remove these two lines without further additions.

Reply:
Forgot these were for atomic policies, so added them back in. Tested with nvcc+gcc.